shortlister/controller.py

```diff
@@ -66,12 +68,31 @@ def edit_criteria_select(self, k=None):
         
     def edit_score_confirm(self,k=None):
         """Confirm changes to score"""
-        
         self.view.view_update(self.current_criterion.name,self.current_criterion.scores[int(k)])
-        update_applicant_score(self.current_applicant,self.current_criterion,int(k))
-        self.view.view_applicant_details(self.current_applicant)
+        update_applicant_score(self.applicant(self.applicant_index),self.current_criterion,int(k))
+        self.view.view_applicant_details(self.applicant(self.applicant_index))
         self.options = self.options_applicant_detail
 
+    def switch_prev_applicant(self,k=None):
+        """shows details of the previous applicant in the shortlist"""
+        if self.applicant_index == 0:
+            pass
+        else:
+            self.applicant_index -=1
+            self.view.view_applicant_details(self.applicant(self.applicant_index))
+
+    def switch_next_applicant(self,k=None):
+        """shows details of the next applicant in the shortlist"""
+        if self.applicant_index+1 >= len(self.shortlist.applicants):
+            self.applicant_index = 0
+        else:
+            self.applicant_index+=1
```

Comment:
It's not very clear what we're doing here (imagine if someone else comes and read the code). There are a two things I recommend. You can add a comment before the condition like

```
# go to the next applicant, or loop back to the first applicant if we're at the end
```

The logic could also be simplified by always incrementing and then looping back if necessary. Makes the intention clear.

```
self.applicant_index += 1
if self.applicant_index > len(self.shortlist.applicants) - 1:
    self.applicant_index = 0
```